script/tests/test_encryption.sh

```diff
@@ -565,6 +610,69 @@ testJWE() {
 	$CTR images rm --sync ${ALPINE_DEC} ${ALPINE_ENC} &>/dev/null
 }
 
+testLocalKeys() {
+        createJWEKeys
+
+	echo "Testing JWE type of encryption with local unpack keys"
```

Comment:
shfmt?

Reply:
good call, slipped my mind, I will add a new PR to add this to travis test as well.